AZ3166/src/libraries/WebSocket/examples/WebSocketEcho/WebSocketEcho.ino

```diff
@@ -0,0 +1,151 @@
+#include "AZ3166WiFi.h"
+#include "WebSocketClient.h"
+
+static bool hasWifi;
+static bool isWsConnected;
+static char *webSocketServerUrl = "ws://devkitwebsocket.azurewebsites.net/test?nickName=test2";
```

Comment:
we maintain this site?

Reply:
Yes, we can provide a simple echo WebSocket server which is hosted on our own subscription

---
In reply to: [173717739](https://github.com/Microsoft/devkit-sdk/pull/771#discussion_r173717739) [](ancestors = 173717739)

Reply:
Use a public websocket echo service instead:
ws://echo.websocket.org
or
ws://demos.kaazing.com/echo

---
In reply to: [173718008](https://github.com/Microsoft/devkit-sdk/pull/771#discussion_r173718008) [](ancestors = 173718008,173717739)